crates/ruma-client-api/src/message/get_message_events.rs

```diff
@@ -35,17 +35,27 @@ pub mod v3 {
 
             /// The token to start returning events from.
             ///
-            /// This token can be obtained from a
-            /// prev_batch token returned for each room by the sync API, or from a start or end token
-            /// returned by a previous request to this endpoint.
+            /// This token can be obtained from a `prev_batch` token returned for each room by the
+            /// sync endpoint, or from a `start` or `end` token returned by a previous request to
+            /// this endpoint.
+            ///
+            /// If this is `None`, the server will return messages from the start or end of the
+            /// history visible to the user, depending on the value of [`dir`].
+            ///
+            /// This field is marked required in v1.2 of the specification, but that is changing,
+            /// most likely as part of v1.3. To frontload the breaking change, this field is
+            /// optional already even though v1.2 is the latest version at the time of writing.
+            /// The specification change can be found [here].
```

Comment:
```suggestion
            /// *Note: This field is marked required in v1.2 of the specification, but that is changing,
            /// most likely as part of v1.3. To frontload the breaking change, this field is
            /// optional already even though v1.2 is the latest version at the time of writing.
            /// The specification change can be found [here].*
```

I think this makes it clear its a special line, not one about what this does, but *why* it is the way it is rn.

Also I wonder if this line is needed at all, if we're going to remove it anyways in 1.3, could this just be a regular non-doc comment?